config/control-plane/kustomization.yaml

```diff
@@ -90,6 +87,26 @@ patches:
 # Note1: Now as the 'patchesJson6902' is deprecated, the direct use of the 'PatchTransformer' is the only way to change a resource namespace to something different from the value configured by the global namespace transformer.
 # Note2: the 'grafana_dashboard' label could be applied using a standard JSON patch, but a namespace must be patched here and it's better to have all related customizations in a single place. 
 transformers:
+- |-
+  apiVersion: builtin
+  kind: NamespaceTransformer
+  metadata:
+    name: add-resources-to-kcp-system
+    namespace: kcp-system
+  unsetOnly: true
```

Comment:
Also related to above comment wrt. `kyma-system` namespace. If we need to keep it, could we still:

Since we are only setting the unset-ones, could the `kyma-system` namespace `PatchTransformers` be replaced by directly setting the namespace and making the rest a regular patch?

Reply:
you mean setting the Kyma-system namespace directly in the ConfigMap? If this is the case, I was trying to do so but couldn't find a way to set a namespace for `ConfigMapGenerator`

Reply:
Yeah good point, I also don't know for sure. There are some examples where they set the namespace in a config map generator. But there also seems to be some caution necessary. In doubt, fine to leave it as is.

https://kubectl.docs.kubernetes.io/references/kustomize/kustomization/configmapgenerator/#overriding-base-configmap-values

Reply:
I tried this earlier but I think this only works for the Kustomization not the Component probably because it produced error when I tried to have it